test/e2e/integration/frontend-test/pdf.ts

```diff
@@ -16,12 +16,13 @@ describe('PDF', () => {
     });
   });
 
-  it('should generate PDF for changename step', () => {
+  it.only('should generate PDF for changename step', () => {
```

Comment:
Whoops, better not commit these! :scream: There's a pre-commit husky script that should warn you when committing a test isolated using `it.only`, did it not work correctly for you or did you skip pre-commit checks? :thinking:

Reply:
I suspect it does not work correctly, I have never been stopped from making this mistake 😅

Reply:
ops, i did not skip it intensionally. Im not sure, havent seen any.